src/effects/effectslot.cpp

```diff
@@ -316,33 +316,36 @@ void EffectSlot::loadParameters() {
 
         m_pControlNumParameters[parameterType]->forceSet(numParameters(parameterType));
 
-        unsigned int iParameter = 0;
+        unsigned int parameterSlotIndex = 0;
         unsigned int numParameterSlots = m_iNumParameterSlots[parameterType];
         for (int i=0 ; i<m_mapForParameterType[parameterType].size() ; ++i) {
```

Comment:
Add a comment above this line: `Load EffectParameters into the slot indicated by m_mapForParameterType`.